x-pack/filebeat/module/fortinet/firewall/manifest.yml

```diff
@@ -0,0 +1,21 @@
+module_version: 1.0
+
+var:
+  - name: syslog_host
+    default: localhost
+  - name: tags
+    default: [fortinet-firewall]
+  - name: syslog_port
+    default: 9001
```

Comment:
This default already collides with other 4 filesets. Can you change it to `9004` which is unused?